.github/sync-files.yaml

```diff
@@ -0,0 +1,8 @@
+- repository: autowarefoundation/autoware
+  files:
+    - source: CONTRIBUTING.md
+    - source: .github/ISSUE_TEMPLATE/config.yml
+    - source: .github/ISSUE_TEMPLATE/task.yaml
+    - source: .github/dependabot.yaml
+    - source: .github/stale.yml
+    - source: .github/workflows/sync-files.yaml
```

Comment:
@BonoloAWF Could you confirm the contents? :pray:

Reply:
Since this repository doesn't have any code, the synced files are limited.
https://github.com/autowarefoundation/autoware_msgs/pull/3/files